### PR TITLE
NXP-19888: fix writing of date properties

### PR DESCRIPTION
**Pull request by afjusto**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo-binary-metadata/pull/5" target="_blank">ReviewNinja

**Comment by atchertchian** [comment]
:+1:

**Comment by nelsonsilva** [comment]
👍

File: src/test/java/org/nuxeo/binary/metadata/test/TestBinaryMetadataService.java [hunk 1]
```diff
@@ -90,6 +95,15 @@
...
         inputPSDMetadata.put("EXIF:Software", "Nuxeo");
         inputPSDMetadata.put("IPTC:Keywords", new String[] {"keyword1", "keyword2"});
+        inputPSDMetadata.put("IPTC:Keywords", new String[] {"keyword1", "keyword2"});
+        try {
+            Date date = new SimpleDateFormat("yyyy:MM:dd HH:mm:ss").parse("2018:06:15 00:00:00");
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(date);
+            inputPSDMetadata.put("EXIF:DateTimeOriginal", calendar);
+        } catch (ParseException e) {
+            fail(e.getMessage());
```
Comment (efge) [inline src/test/java/org/nuxeo/binary/metadata/test/TestBinaryMetadataService.java:105]: Why this form of catch/fail? Why not simply let the exception bubble up to the test framework, if it normally isn't produced?

Comment (afjusto) [reply src/test/java/org/nuxeo/binary/metadata/test/TestBinaryMetadataService.java:105]: Just took as an example a test that was already in place (`itShouldAcceptQuoteInMetadataAndAllASCII`). 

Will throw a new `IllegalArgumentException` instead.

Comment (efge) [reply src/test/java/org/nuxeo/binary/metadata/test/TestBinaryMetadataService.java:105]: No just leave out the whole try/catch thing. Just execute the statements if they're supposed to work as is.

**Pull request closed without merging**